js/scripts/release.sh

```diff
@@ -89,11 +89,13 @@ fi
 
 echo "*** Updating cargo parity-ui-precompiled#$PRECOMPILED_HASH"
 git submodule update
+sed -i "/^parity-ui-precompiled/ { s/branch = \".*\"/branch = \"$BRANCH\"/g; }" dapps/ui/Cargo.toml
```

Comment:
With ' the shell doesn't expand the variables, with " it does. (Hence the escape jiggle)